atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/CassandraMultinodeEteTest.java

```diff
@@ -0,0 +1,228 @@
+/**
```

Comment:
Are you sure this test gets run? `build.gradle` is set up to only run TestSuites, and I don't see this added as part of a suite.